generator/src/main/java/org/apache/kafka/message/StructRegistry.java

```diff
@@ -110,6 +110,7 @@ private void addStructSpecs(Versions parentVersions, List<FieldSpec> fields) {
                     // Synthesize a StructSpec object out of the fields.
                     StructSpec spec = new StructSpec(typeName,
                         field.versions().toString(),
+                        null, // version deprecations not supported at field level //FIXME Perhaps we should pass `Versions.NONE.toString` instead
```

Comment:
Do you intend to address the FIXME in this patch?

Reply:
Yes! My bad for not fixing this before I asked for the review. I attempted a fix, let's see if any tests fail.